app/v1/models/response/retrieveBusinessDetails/des/BusinessDetails.scala

```diff
@@ -53,11 +54,24 @@ case class BusinessDetails(businessId: String,
 }
 
 object BusinessDetails {
-  implicit val reads: Reads[BusinessDetails] = (
+
+  val cashOrAccrualsReads: Reads[AccountingType] = (JsPath \ "cashOrAccrualsFlag").read[Boolean].map {
+    case false => CashOrAccruals.`cash`
+    case true => CashOrAccruals.`accruals`
+  }.map(_.toMtd)
+
+  implicit val accountingPeriodReads: Reads[Seq[AccountingPeriod]] = (
+    (JsPath \ "accountingPeriodStartDate").read[String] and
+      (JsPath \ "accountingPeriodEndDate").read[String]
+    )apply((start, end) =>  Seq(AccountingPeriod(start, end)))
+
+  implicit val writes: OWrites[BusinessDetails] = Json.writes[BusinessDetails]
+
+  implicit val readsBusinessData: Reads[BusinessDetails] = (
```

Comment:
Again, you're only using this explicitly so there's no reason for it to be implicit